src/elastic.js

```diff
@@ -0,0 +1,38 @@
+const { ELASTIC_URL, ELASTIC_PORT } = process.env;
+const { Client } = require('@elastic/elasticsearch');
+const Mock = require('@elastic/elasticsearch-mock');
+
+function MockClient(options) {
+  const mock = new Mock();
+  // Mock out various responses we'll need:
+  mock.add(
+    {
+      method: ['PUT', 'POST', 'GET', 'DELETE'],
+      path: '*',
+    },
+    () => {
+      return { status: 'ok' };
+    }
+  );
+
+  const client = new Client(options);
```

Comment:
I think you need to do what the mock client does with Elastic to set the mock connection on options:

```js
const { Client } = require('@elastic/elasticsearch')
const Mock = require('@elastic/elasticsearch-mock')
 
const mock = new Mock()
const client = new Client({
  node: 'http://localhost:9200',
  Connection: mock.getConnection()
})
```